adapters/Vungle/VungleAdapter/VungleRouter.h

```diff
@@ -2,7 +2,7 @@
 #import <VungleSDK/VungleSDK.h>
 #import "VungleAdNetworkExtras.h"
 
-typedef NS_ENUM(NSUInteger, VungleNetworkAdapterAdType) { Unknown, Rewarded, Interstitial, MREC };
+typedef NS_ENUM(NSUInteger, VungleNetworkAdapterAdType) { Unknown, Rewarded, Interstitial, MREC, Banner, ShortBanner, LeaderboardBanner };
```

Comment:
Mostly a format recommendation - I'd show each on it's own row (as `BannerRouterDelegateState` below) for readability.

Reply:
@crbinsf Hi Clarke, got it, I will modify the ENUM type format.  :)